src/coreclr/jit/simd.h

```diff
@@ -53,7 +53,9 @@ struct simd8_t
         return {};
     }
 };
+static_assert_no_msg(sizeof(simd8_t) == 8);
 
+#include "pshpack4.h"
```

Comment:
`poppack.h` is included with the `<>` syntax. Which is the right one?

Reply:
thanks, normalized to <>